src/polymer/polymer2-element-scanner.ts

```diff
@@ -70,6 +70,10 @@ class ElementVisitor implements Visitor {
       const namespacedClassName =
           getNamespacedIdentifier(className, nodeJsDocs);
       element.className = namespacedClassName;
+
+      const summaryTag = jsdoc.getTag(nodeJsDocs, 'summary');
+      element.summary = summaryTag && summaryTag.description || '';
```

Comment:
`(summaryTag && summaryTag.description) || ''`
I know what you have works correctly but it helps to be explicit here